### PR TITLE
[PLAT-30] Bundle the Add-On manager OWA within the Platform

### PR DESCRIPTION
**Pull request by Elbertbiggs360**
### JIRA TICKET NAME
[[PLAT-30] Bundle the Add-On manager OWA within the Platform](https://issues.openmrs.org/browse/PLAT-30)
## Description of what I changed
Modified the `Index.htm` to provide a link to the Add On manager OWA

## Issue I worked on
* [[PLAT-30]](https://issues.openmrs.org/browse/PLAT-30)
## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by Charpell** [comment]
Looks good to me

File: webapp/src/main/webapp/index.htm [hunk 1]
```diff
@@ -38,11 +38,18 @@
...
-        				<p>If you are a developer, you can access the REST API. (See <a href="https://wiki.openmrs.org/x/P4IaAQ">REST documentation for clients</a>)</p>
-        			</div>
+                    <div class="bar">
+                        <h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
+                        <p>If you are seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module
+                            is installed.</p>
+                        <p>
+                            <a href="/openmrs/owa/openmrs-addonmanager/index.html#/"><em>Click here</em></a> to access Add-on manager and install modules</p>
+                        <p> To learn more about the available User Interface Modules, visit
+                            <a href="https://wiki.openmrs.org/x/d4aIBQ" target="_blank">this link</a>
```
Comment (dkayiwa) [inline webapp/src/main/webapp/index.htm]: What is the meaning of "this link"?

File: webapp/src/main/webapp/index.htm [hunk 2]
```diff
@@ -38,11 +38,18 @@
...
-        				<h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
-        				<p>If you are seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module is installed. Learn about the available <a href="https://wiki.openmrs.org/x/d4aIBQ">User Interface Modules</a></p>
-        				<p>If you are a developer, you can access the REST API. (See <a href="https://wiki.openmrs.org/x/P4IaAQ">REST documentation for clients</a>)</p>
-        			</div>
+                    <div class="bar">
+                        <h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
+                        <p>If you are seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module
+                            is installed.</p>
+                        <p>
+                            <a href="/openmrs/owa/openmrs-addonmanager/index.html#/"><em>Click here</em></a> to access Add-on manager and install modules</p>
```
Comment (dkayiwa) [inline webapp/src/main/webapp/index.htm]: For consistency, can we use Add On Manager?

Comment (Elbertbiggs360) [reply webapp/src/main/webapp/index.htm]: Alright

File: webapp/src/main/webapp/index.htm (hunk 1 above)
Comment (Elbertbiggs360) [reply webapp/src/main/webapp/index.htm]: I get it. Let me give a more clear description of what is at that link

File: webapp/src/main/webapp/index.htm [hunk 3]
```diff
@@ -38,11 +38,18 @@
...
-        				<h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
-        				<p>If you are seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module is installed. Learn about the available <a href="https://wiki.openmrs.org/x/d4aIBQ">User Interface Modules</a></p>
-        				<p>If you are a developer, you can access the REST API. (See <a href="https://wiki.openmrs.org/x/P4IaAQ">REST documentation for clients</a>)</p>
-        			</div>
+                    <div class="bar">
+                        <h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
+                        <p>If you are seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module
+                            is installed.</p>
+                        <p>
+                            <a href="/openmrs/owa/openmrs-addonmanager/index.html#/"><em>Click here</em></a> to access Add On manager and install modules</p>
```
Comment (dkayiwa) [inline webapp/src/main/webapp/index.htm]: Didn't we agree that manager should be capital M?

Comment (Elbertbiggs360) [reply webapp/src/main/webapp/index.htm]: Yes we did. I am going to address that change after confirming the link works locally. Thanks for the heads up @dkayiwa 🙂

File: webapp/src/main/webapp/index.htm [hunk 4]
```diff
@@ -38,11 +38,18 @@
...
-        			<div class="bar">
-        				<h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
-        				<p>If you are seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module is installed. Learn about the available <a href="https://wiki.openmrs.org/x/d4aIBQ">User Interface Modules</a></p>
-        				<p>If you are a developer, you can access the REST API. (See <a href="https://wiki.openmrs.org/x/P4IaAQ">REST documentation for clients</a>)</p>
-        			</div>
+                    <div class="bar">
+                        <h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
+                        <p>If you are seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module
+                            is installed.</p>
+                        <p>
```
Comment (dkayiwa) [inline webapp/src/main/webapp/index.htm]: "Click here to access Add On Manager". Is this grammar correct?

Comment (Elbertbiggs360) [reply webapp/src/main/webapp/index.htm]: Updated.
Is "Click here to access **the** Add On Manager" a better sentence construct?

Comment (dkayiwa) [reply webapp/src/main/webapp/index.htm]: Exactly!

**Comment by mozzy11** [comment]
The reason for the revert was never mentioned in the ticket Comments.

**Pull request merged**